apps/ledger-live-desktop/src/renderer/screens/manager/DeviceStorage/index.jsx

```diff
@@ -30,6 +31,8 @@ import nanoSPDark from "~/renderer/images/devices/nanoSP_dark.png";
 import nanoX from "~/renderer/images/devices/nanoX.png";
 import nanoXDark from "~/renderer/images/devices/nanoX_dark.png";
 import blue from "~/renderer/images/devices/blue.png";
+import { Flex } from "@ledgerhq/react-ui";
```

Comment:
oh i think the linter does not see it because it's in a jsx file, but it should be before other local imports